src/server/rpc/procedures/MovieDB/MovieDB.js

```diff
@@ -0,0 +1,358 @@
+// This is the MovieDB game RPC. It wraps the web API of themoviedb.org.
+
+'use strict';
+
+var mdb = require('moviedb')(process.env['TMDB_API_KEY']),
+    debug = require('debug'),
+    request = require('request'),    
+    CacheManager = require('cache-manager'),   
+    cache = CacheManager.caching({store: 'memory', max: 1000, ttl: Infinity}),
+    info = debug('NetsBlox:RPCManager:NPlayer:info'),
+    trace = debug('NetsBlox:RPCManager:MovieDB:trace');
+
+
+// Retrieving static images
+var baseUrl = 'https://image.tmdb.org/t/p/w500';
+
+/**
+ * MovieDB - This constructor is called on the first request to an RPC
+ * from a given room
+ *
+ * @constructor
+ * @return {undefined}
+ */
+var MovieDB = function() {};
```

Comment:
Since this is empty, it seems that a static RPC may be a better fit (and example can be found in https://github.com/NetsBlox/NetsBlox/blob/master/src/server/rpc/procedures/EarthQuakes/EarthQuakes.js)